opslevel/common.go

```diff
@@ -270,6 +270,22 @@ func flattenTagArray(tags []opslevel.Tag) []string {
 	return output
 }
 
+func flattenMembersArray(members []opslevel.UserId) []string {
+	output := []string{}
+	for _, member := range members {
+		output = append(output, member.Email)
+	}
+	return output
+}
+
+func flattenTeamsArray(teams []opslevel.TeamId) []string {
+	output := []string{}
+	for _, team := range teams {
+		output = append(output, team.Alias)
```

Comment:
This should flatten to an ID - not an Alias since the result is going into the state.

Reply:
Went with Alias instead of ID as discussed since we couldn't do both